hack/verify-codegen.sh

```diff
@@ -17,51 +18,7 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-if [[ -n "${TEST_WORKSPACE:-}" ]]; then # Running inside bazel
-  echo "Checking generated clients for changes..." >&2
-elif ! command -v bazel &>/dev/null; then
-  echo "Install bazel at https://bazel.build" >&2
-  exit 1
-else
-  (
-    set -o xtrace
-    bazel test --test_output=streamed //hack:verify-codegen
-  )
-  exit 0
-fi
-
-tmpfiles=$TEST_TMPDIR/files
-
-(
-  mkdir -p "$tmpfiles"
-  rm -f bazel-*
-  cp -aL "." "$tmpfiles"
-  # clean up 'external' directory copied from test runfiles
-  rm -rf "$tmpfiles"/external
-  export BUILD_WORKSPACE_DIRECTORY=$tmpfiles
-  export HOME=$(realpath "$TEST_TMPDIR/home")
-  unset GOPATH
-  go=$(realpath "$2")
-  export PATH=$(dirname "$go"):$PATH
-  "$@"
-)
-
-(
-  # Remove the platform/binary for gazelle and kazel
-  gazelle=$(dirname "$3")
-  kazel=$(dirname "$4")
-  rm -rf {.,"$tmpfiles"}/{"$gazelle","$kazel"}
-)
-# Avoid diff -N so we handle empty files correctly
-# Ignoring Copyright to not trigger a CI fail every new year
-diff=$(diff -upr \
-  -I '^Copyright.*' \
-  "./pkg" "$tmpfiles/pkg" 2>/dev/null || true)
+echo "!!! hack/verify-codegen.sh is now a no-op and will soon be removed" 2>&1
+echo "!!! use "make verify-codegen" instead" 2>&1
 
-if [[ -n "${diff}" ]]; then
-  echo "${diff}" >&2
-  echo >&2
-  echo "ERROR: generated clients changed. Update with ./hack/update-codegen.sh" >&2
-  exit 1
-fi
-echo "SUCCESS: generated clients up-to-date"
+exit 0
```

Comment:
This could definitely lead to false test-passes if someone has an existing script or pipeline that invokes verify-codegen :)

Reply:
My main focus was on the main cert-manager CI pipeline - if this calls `make verify-codegen` then we end up running that twice currently since we're running both  pull-cert-manager-bazel and pull-cert-manager-make-test

I'll just make it run "make verify-codegen" and the inefficiency be damned; it'll be good to at least have another reason to make us go faster ripping the bazel tests out.